### DISCUSSION TITLE
loading from pre-MC environments & ll.blues dest_acts

### DISCUSSION THREAD

**Discussion by the-steech** [ppooll-dev/ppooll#92] [General]
got some older environments that i'm trying to get updated for the new MC ll.blues configuration.  

maybe there is a way to parse the old "dest_acts" & "dest_channels" params to set the new "outputs" param?  @klausfilip could you point me to where I could start to tinker within ho_st?  (worse case scenario i can use the old envi files to manually reset all ll.blues menus, but some of my environments are massive)

old environment ll.blues params:
`
"ll.blues" : {
			"state" : [ 1, 1, 1, 1, 0, 0, 1 ],
			"dest_channels" : [ "out_l", "out_r", "in_4.1", "in_4.2" ],
			"dest_acts" : [ "ho_st1", "ho_st1", "matriarch@1", "matriarch@1" ],
			"levels" : [ 0.0, 0.0, 1.0, 1.0, 10.0, 0.0, 0.5 ]
		}
`

new environment ll.blues params:
`
"ll.blues" : {
			"state" : [ 1, 1, 1, 1, 1, 0, 1 ],
			"outputs" : [ "ho_st1~out.1", "no~-no-" ],
			"levels" : [ 0.6, 0.6, 1.0, 1.0, 10.0, 0.0, 0.5 ]
		}
`

**Comment by klausfilip** [comment]
hmm, this is a non trivial problem.
we don't know the destination input-names.
but here is a crude hack that maybe works in 80%..
